src/App.tsx

```diff
@@ -1,9 +1,67 @@
 import './App.scss';
+import {
+  Routes, Route, NavLink, Navigate,
+} from 'react-router-dom';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFound } from './components/NotFound';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App = () => {
+  return (
+    <div className="App">
+      <div className="tabs is-centered">
+        <ul>
+          <li>
+            <NavLink
+              to="/"
+              style={({ isActive }) => {
+                return {
+                  color: isActive ? 'blue' : '',
+                };
+              }}
```

Comment:
I believe using `classnames` would be a better solution instead of applying ternary with an empty string